README.md

```diff
@@ -1 +1,175 @@
-# Budget-App
\ No newline at end of file
+<a name="readme-top"></a>
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+    - [Live demo](#🚀live-demo)
+    - [Presentation](#presentation)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Budget App] <a name="about-project"></a>
+
+💰💼 Budget App is a web-based budgeting application built using Ruby on Rails and PostgreSQL. 
+With BudgetApp, users can effortlessly track their income and expenses, gaining full control over their finances.
+
+## 🛠 Built With <a name="built-with"></a>
+
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">Rails</a></li>
+  </ul>
+</details>
+
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- Gitflow execution
+- Register an account
+- Login registered account
+- Create categories and transactions
+- Upload Profile photo
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### 🚀Live Demo <a name="Live Demo"></a>
+
+[Live]()
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Presentation <a name="Presentation"></a>
```

Comment:
Good job working on the project requirements 😄 But you did not add a deploy link for your project in your readme file it says in the project requirements that:



The student deploys the application using Render, Netlify or GitHub Pages.
The project is deployed properly, the home and detail page works as expected.
A link to the deployed project is present in the README.
So it would be nice if you implemented this.